<ui version="4.0" >
 <class>GlobalInstrumentViewBase</class>
 <widget class="QWidget" name="GlobalInstrumentViewBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>-1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>-1</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>-1</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Midi input key:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="inKeySpinBox" >
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="maximum" >
          <number>127</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addOutputButton" >
         <property name="text" >
          <string>Add output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeOutputButton" >
         <property name="text" >
          <string>Remove output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editOutputButton" >
         <property name="text" >
          <string>Edit output</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Midi output definitions:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="outputListWidget" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
